hardware/Makefile

```diff
@@ -92,13 +92,14 @@ $(buildpath):
 # Bender
 bender:
 	@[ -x ./bender ] && echo "Bender already exists." || \
-	(cat $(ARA_DIR)/scripts/bender-init | bash -s -- 0.21.0)
+	curl --proto '=https' --tlsv1.2 https://fabianschuiki.github.io/bender/init -sSf | sh	-s -- 0.23.1
 	@echo "$$(./bender --version) available."
 
 # Patches
 .PHONY: apply-patches
 apply-patches: patches
 	cd deps/tech_cells_generic && git apply ../../patches/0001-tech-cells-generic-sram.patch
+	cd deps/cva6 && git apply ../../patches/0002-cva6-cache-size.patch
```

Comment:
Not sure this change would ever get merged into the upstream CVA6, though, since it is a choice that depends on our backend trials. That is why I wanted to keep it in our local repo, even if this was done through a patch.

Reply:
It makes sense since it would be good to converge and get back to the upstream CVA6, in the end. The only problem with patches added after the initial clone is that they will not be applied by users.

Reply:
I agree. This change only hurts if you are PnRing the design, however, and if that is the case you might want to change the cache size to another value.